cmd/sdk-server/main.go

```diff
@@ -65,7 +65,7 @@ var (
 
 func main() {
 	ctlConf := parseEnvFlags()
-	logger.WithField("version", pkg.Version).
+	logger.WithField("version", pkg.Version).WithField("flags", runtime.EncodeFeatures()).
```

Comment:
Why log the feature gates as "flags" instead of "featureGates" (or "feature-gates")?

(same thing in the other cmd/.../main.go files).

Reply:
Yeah, that's a completely valid point. Making the change now.

`featureGates` seems most consistent with the other names.